messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/MessagingTransformerRegistration.java

```diff
@@ -58,12 +60,19 @@ public String getSubsystemName() {
     @Override
     public void registerTransformers(SubsystemTransformerRegistration registration) {
         ChainedTransformationDescriptionBuilder builder = TransformationDescriptionBuilder.Factory.createChainedSubystemInstance(registration.getCurrentSubsystemVersion());
+        registerTransformers_WF_36(builder.createBuilder(MessagingExtension.VERSION_17_0_0, MessagingExtension.VERSION_16_0_0));
```

Comment:
nitpick: this method would be more correct if it is named as `registerTransformers_WF_35` instead, since the transformer chain here is against VERSION_16_0_0, which was shipped in WF35